uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/BAO/Document.php

```diff
@@ -19,8 +19,8 @@ public static function create(&$params) {
 
     if ($hook === 'create') {
       // If creating a new Document, we require all three contacts to be defined.
```

Comment:
this inline comment is not longer reflect the current behavior , a better one could be : 

// If creating a new Document, both target_contact_id and source_contact_id are required